lang/hi-in.js

```diff
@@ -0,0 +1,125 @@
+/* eslint quotes: 0 */
```

Comment:
Is this a new language we are supporting?

Also, looks like the tag translations were missed for `hi`.

Reply:
Looks like it was an issue with serge https://d2l.slack.com/archives/C0TF4KSGZ/p1666719158315249

Reply:
Thanks Devon.

@oelhanafey , so for our translation updates, we'll just need to copy the following translations out of `hi-in.js` into out `hi.js` and then remove `hi-in.js`. This file was created in error due to a config issue that should be corrected for next time, but we still want those two translations.

```
"components.tag-list.role-description": "टैग सूची",
"components.tag-list-item.role-description": "टैग",
```

Reply:
> @oelhanafey , so for our translation updates, besides our regular updates, we'll need to copy the following translations out of `hi-in.js` into our `hi.js` and then remove `hi-in.js`.

Okay sounds good thanks. I'll copy those over